getting-started/installation-apt-debian.md

```diff
@@ -1,6 +1,6 @@
->:WARNING: Our scaling out capabilities are currently in BETA and
-are not available through this installation method. For more information,
-please [contact us][contact] or join the #multinode-beta channel in our 
+>:WARNING: These installation instructions will install the current Release Candidate
+for TimescaleDB 2.0, which includes the ability to setup [Multi-Node capabilities][multi-node-basic]. For
```

Comment:
Minor nit:  I think we've generally been keeping "multi-node" lower case, just for consistency.